### DISCUSSION TITLE
[PandaStudio] Add `BaseURL` configuration entry

### DISCUSSION THREAD

**Discussion by theMasix** [chakra-ui/panda#2401] [Ideas]
### Description

It's assumed that the user wants to deploy the Panda Studio at the root URL (e.g `studio.mydesignsystem.com/`)

A base URL should be configurable to deploy the Panda Studio at non-root base urls.

### Problem Statement/Justification

With this assumption, all redirection appears to point to the root level of URL and start with `/` address.

### Proposed Solution or API

add a `baseURL`, `homepage` or `basePath` to Panda Studio config

**Comment by segunadebayo** [comment]
You can configure this by setting the `BASE_URL` environmental variable.

> We could consider adding this to the studio options directly.
